fpdf/fpdf.py

```diff
@@ -361,6 +364,23 @@ def __init__(
         # final buffer holding the PDF document in-memory - defined only after calling output():
         self.buffer = None
 
+    def set_encryption(
+        self,
+        owner_password,
+        user_password=None,
+        encryption_method=None,
+        permissions=AccessPermission.all(),
+        encrypt_metadata=False,
+    ):
```

Comment:
Could you please add a doctring?

Reply:
I want to thank you for your patience. It's my first PR and I'm learning a lot :D

Reply:
Wow!
Well you are doing very well for you 1st PR, congratulations!

And "Joyeux Noël" as we say here in France 😊

Reply:
On dit pareil au Québec! Joyeyx Noël et bonne année